website/package.json

```diff
@@ -2,26 +2,22 @@
   "scripts": {
     "start": "node server/server.js",
     "test": "node server/generate.js",
-    "gh-pages": "node publish-gh-pages.js",
-    "crowdin-upload": "crowdin-cli --config ../crowdin.yaml upload sources --auto-update -b master",
-    "crowdin-download": "crowdin-cli --config ../crowdin.yaml download -b master"
+    "gh-pages": "node publish-gh-pages.js"
   },
   "dependencies": {
-    "classnames": "2.2.4",
     "connect": "2.8.3",
     "feed": "0.3.0",
     "fs.extra": "*",
     "glob": "*",
     "mkdirp": "*",
     "object-assign": "^2.0.0",
     "optimist": "0.6.0",
-    "prettier": "^1.2.2",
```

Comment:
This is needed. I use prettier to format some generated files in one of the steps.